components/Discussion/Comments.js

```diff
@@ -83,6 +99,14 @@ class Comments extends PureComponent {
           }
         })
     }
+
+    this.toggleShare = (path, commentId) => {
```

Comment:
since you're not really toggling here I'd just use `setState` directly, see suggestion